magento/sales.py

```diff
@@ -33,6 +33,26 @@ def list(self, filters=None):
         """
         return self.call('sales_order.list', [filters])
 
+    def search(self, options=None):
```

Comment:
instead of options, take each argument separately and construct the dictionary inside